pkg/cmd/promhelperservice/handlers/instanceconfigs/helpers.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2024 The Cockroach Authors.
+
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package instanceconfigs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+)
+
+type ResultResponse string
+
+const (
+	// resourceName and version for the API
+	resourceName    = "instance-configs"
+	resourceVersion = "v1"
+	// ClusterIDParam is the url parameter
+	ClusterIDParam = "cluster_id"
+	// defaultInstanceConfigDir is the default instance config directory from where the yml files are created/removed.
+	defaultInstanceConfigDir = "/opt/prom/prometheus/instance-configs"
+	instanceConfigFileExt    = "yml"
+	// instanceConfigEnvVar needs to be set to override the defaultInstanceConfigDir
+	instanceConfigEnvVar = "INSTANCE_CONFIG_DIR"
+
+	// Result responses
+	SUCCESS ResultResponse = "success"
+	INVALID ResultResponse = "invalid"
+	FAILED  ResultResponse = "failed"
+)
+
+// FormInstanceConfigUrl returns the POST url for instance config
+func FormInstanceConfigUrl() string {
+	return fmt.Sprintf("%s/%s", resourceVersion, resourceName)
+}
+
+// FormInsConfUrlWithClusterID returns the url with cluster ID for instance config
+func FormInsConfUrlWithClusterID() string {
```

Comment:
Actually, this was named this way thinking if we ever want to imeplement GET, PUT or PATCH, this will use `FormInsConfUrlWithClusterID`. Similarly, for LIST, we would need `FormInstanceConfigUrl`

Reply:
I think the main goal of this service is to stay simple. I hope we will never need to implement GET, PUT or PATCH. If, for some unfortunate reason, we end up needing that functionality, then we can see if we need better names.

I was also quite confused by the name initially, as it took me a while to understand that `Ins` refers to "instance" (config?) I'm still not sure I understand what `Form` is supposed to represent.

Reply:
Sure, let me rename it. "Form" = create/generate